src/components/view/card.vue

```diff
@@ -108,6 +127,30 @@ export default {
             break
         }
       }
+    },
+    // returns the current time interval from an end date to start date
+    currentTimeInterval: function () {
+      return this.$store.getters[this.path + '/dateEnd'] - this.$store.getters[this.path + '/dateStart']
+    },
+    // returns boolean for if next interval exists in program.
+    nextExists: function () {
+      return this.nextEndpoint < Date.now()
+    },
+    // returns boolean for if prev interval exists in program
+    prevExists: function () {
+      // Check if user is asking for data in the past,
+      // since the earliest record in energy_data right now is from 2018
+      // this is the hard-coded in time-stamp and may be subject to change
+      // depending on how far back our records go.
+      return this.nextStartpoint > 1527836400000

```

Comment:
We may discuss importing some historical data in the future. Good, descriptive comment.